docs/topics/testing/tools.txt

```diff
@@ -458,6 +458,12 @@ Specifically, a ``Response`` object has the following attributes:
             >>> response.context['name']
             'Arthur'
 
+        .. note::
+          This attribute is only populated when using the Django Template
+          Language. If you use another template engine,
+          :attr:`~django.template.response.SimpleTemplateResponse.context_data`
+          is a suitable alternative.
```

Comment:
Is it worth pointing out that not all responses come from `SimpleTemplateResponse` so they may not have these attributes?